ansible/playbooks/deploy-cinder-volumes-reference.yaml

```diff
@@ -123,59 +124,65 @@
         group: root
         mode: "0644"
       loop:
-        - src: "cinder.conf"
-          dest: /etc/cinder/cinder.conf
-        - src: "backends.conf"
-          dest: /etc/cinder/backends.conf
         - src: "logging.conf"
           dest: /etc/cinder/logging.conf
         - src: "volume.filters"
           dest: /etc/cinder/volume.filters
       notify:
         - Restart cinder-volume systemd services
 
-    - name: Replace the host in the cinder.conf with the current Ansible FQDN
+    - name: Create the cinder-volume configuration stage file
+      changed_when: false
+      ansible.builtin.copy:
+        content: "{{ _kubernetes_cinder_etc_secret.resources[0].data['cinder.conf'] | b64decode }}"
+        dest: "/etc/cinder/cinder.conf.stage"
+        owner: root
+        group: root
+        mode: "0644"
+
+    - name: Replace the host in the cinder.conf with the current Ansible FQDN in the stage file
```

Comment:
cinder.conf should reference cinder.conf.stage in the name section

Reply:
done.